ModFrmHilD/Creation/Weight1.m

```diff
@@ -19,7 +19,7 @@ intrinsic Weight1CuspBasis(
   chiinv := chi^(-1);
   ZF := Integers(M);
 
-  assert IsCompatibleWeight(chi, k);
+  // assert IsCompatibleWeight(chi, k);
```

Comment:
given that you changed `IsCompatibleWeight` does it still make sense to comment this line?